pkgs/tools/X11/wmutils-opt/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, libxcb }:
+
+stdenv.mkDerivation rec {
+  name = "wmutils-opt-${version}";
+  version = "1.0";
+
+  src = fetchurl {
```

Comment:
That sounds good!
Should I add that in another commit or change the existing one?